README.md

```diff
@@ -76,6 +76,7 @@ The format for an individual data channel setting is `<data channel name>=<value
 | EPOCH_LIFETIMES | `typical=3` | No | The amount of current & recent previous epochs considered to be 'active'. Epochs older than this lifetime will be consider 'expired', and all partial measurements will be reported at the end of aggregation, if any.  |
 | EPOCH_DATE_FIELD_NAMES | `typical=wos` | No | The name of the date fields to inject into the aggregated measurements. The injected field will include the survey date, inferred via the measurement epoch. |
 | RANDOMNESS_INSTANCE_NAMES | `typical=typical` | No | Randomness server instance names, for retrieving relevant server info. |
+| MIN_CHANNEL_REVISIONS | | No | The minimum `Brave-Constellation-Rev` header value for measurements submitted to the server. |
```

Comment:
It looks like this takes an `instance_name=value,...` sequence like the other plural env variables. Would be more clear to have"for each channel" or similar in the help text to reflect this.

Reply:
hopefully this is sufficient:
https://github.com/brave/constellation-processors/blob/6f7e18efc814e6d037486d9d700933347073177e/README.md?plain=1#L66-L68

Reply:
As I said, it would be more clear if the inline help mentioned it was per-channel.

Reply:
Oh, yeah, sorry. Look at the surrounding context it's fine.